requirements.txt

```diff
@@ -20,8 +20,6 @@ Pillow > 10.1.0
 mkdocs-open-in-new-tab
 mkdocs-print-site-plugin
 # To update the packages, run the following command in your terminal:
-# pip install --upgrade -r requirements.txt
-# To freeze the current packages and their versions, run the following command in your terminal:
-# pip freeze > requirements.txt
+# pip install --upgrade -r <plugin_name>
```

Comment:
Why plugin name? When you pass the requirements.txt file, it installs all packages listed so you don't have to install them one by one

Reply:
Yes good idea